PhpAmqpLib/Connection/AMQPConnectionFactory.php

```diff
@@ -101,7 +99,8 @@ private static function getSslOptions(AMQPConnectionConfig $config): array
             'verify_peer_name' => $config->getSslVerifyName(),
             'passphrase' => $config->getSslPassPhrase(),
             'ciphers' => $config->getSslCiphers(),
-            'security_level' => $config->getSslSecurityLevel()
+            'security_level' => $config->getSslSecurityLevel(),
+            'crypto_method' => $config->getSslCryptoMethod()
```

Comment:
Please add comma to the end. Let's avoid 2 line changes because of arrays in the future.